### ISSUE TITLE
Add this troubleshooting steps for mounting solbuild image

### ISSUE THREAD

**Issue by malfisya** [getsolus/help-center-docs#553]
sudo mkdir tmp
sudo mount /var/lib/solbuild/images/unstable-x86_64.img tmp
sudo mount --types proc /proc tmp/proc
sudo chroot tmp eopkg it https://cdn.getsol.us/repo/unstable/g/glibc/glibc-2.39-117-1-x86_64.eopkg
sudo umount tmp/proc
sudo umount tmp
sudo rm -r tmp

**Comment by TraceyC77** [comment]
@malfisya - can you briefly describe why someone would need to do this? I can add this, as long as I understand the context

**Comment by TraceyC77** [comment]
Ping @malfisya

**Comment by malfisya** [comment]
I don't think this tip will be useful afterall. I think this happened to me (and a few others) when we had problem with the first attempt of usr-merge. Now, that usr-merge is done, I don't think anyone will face this again.
